src/main/java/org/opensearch/performanceanalyzer/commons/event_process/EventDispatcher.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.performanceanalyzer.commons.event_process;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class EventDispatcher {
```

Comment:
The `EventDispatcher` class should be part of the RCA codebase, any reason we have included it here?

Reply:
'EventDispatcher' and 'EventProcessor' are direct dependencies of 'EventLogFileHandler' that is to come in next PR. They're here now as they don't have tests on their own and not to overwhelm future PR's which may be hardver to review.

Reply:
Thanks for clarifying this @Tjofil , I looked deeper in the code, and it makes sense to move `EventLogFileHandler` and the common Event classes to this repo.